src/libs/OptionsListUtils.js

```diff
@@ -636,7 +637,57 @@ function hasEnabledOptions(options) {
 }
 
 /**
- * Build the options for the category tree hierarchy via indents
+ * Sorts categories
```

Comment:
It would be good to explain the sorting logic a bit, since it's not clear what it's being sorted by

Reply:
Agree, clarifying...